packages/eslint-plugin/ROADMAP.md

```diff
@@ -1,10 +1,10 @@
 ﻿# Roadmap
 
-✅ (28) = done  
-🌟 (79) = in ESLint core  
-🔌 (33) = in another plugin  
-🌓 (16) = implementations differ or ESLint version is missing functionality  
-🛑 (70) = unimplemented
+✅ (29) = done
```

Comment:
an you restore white-spaces?
https://github.com/typescript-eslint/typescript-eslint/blob/c89abbdea5298fccebca4e66fc362037a2ea1fec/packages/eslint-plugin/ROADMAP.md

i think we should convert them to `<br>`, and same below